server/resp/src/main/java/org/infinispan/server/resp/Resp3Handler.java

```diff
@@ -122,6 +122,15 @@ public static void handleBulkResult(CharSequence result, ByteBufPool alloc) {
       }
    }
 
+   public static void handleBulkAsciiResult(CharSequence result, ByteBufPool alloc) {
+      if (result == null) {
+         handleNullResult(alloc);
+      } else {
+         ByteBufferUtils.stringToByteBufAscii("$" + result.length() + CRLF_STRING + result + CRLF_STRING,
+               alloc);
```

Comment:
Should we write piece by piece instead?

Reply:
Yes, we should. The breaking up I did in a separate pass and forgot about this and the handleBulkResult method.